operator/v1/types_network.go

```diff
@@ -788,3 +815,24 @@ const (
 	// between pods on the cluster network.
 	IPsecModeFull IPsecMode = "Full"
 )
+
+// AdvancedRoutingProvider is a component providing advanced routing capabilities.
+// +kubebuilder:validation:Enum=FRR
+type AdvancedRoutingProvider string
+
+const (
+	// AdvancedRoutingProviderFRR determines FRR is providing advanced routing
+	// capabilities.
+	AdvancedRoutingProviderFRR AdvancedRoutingProvider = "FRR"
+)
+
+// AdvancedRouting describes components and relevant configuration providing
+// advanced routing capabilities.
+type AdvancedRouting struct {
+	// providers of advanced routing capabilities.
+	// +listType=set
+	// +kubebuilder:validation:Required
+	// +kubebuilder:validation:MinItems=1
+	// +required
```

Comment:
If you remove this, does it change anything?

Reply:
It looks like it doesn't make a difference, removed.